.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
```

Comment:
Travis detects if you have a yarn.lock file, and if it is the case, it installs yarn by default